solutions/instance/variables.tf

```diff
@@ -56,3 +56,57 @@ variable "access_tags" {
   description = "A list of access tags to apply to the Object Storage instance created by the module. [Learn more](https://cloud.ibm.com/docs/account?topic=account-access-tags-tutorial)."
   default     = []
 }
+
+variable "existing_secrets_manager_instance_crn" {
+  type        = string
+  default     = null
+  description = "The CRN of existing secrets manager to use to create service credential secrets for COS instance."
+}
+
+variable "existing_secrets_manager_endpoint_type" {
+  type        = string
+  default     = "public"
+  description = "The type of endpoint to use for communicating with the Secrets Manager instance. Possible values: `public`, `private`. Ignored if `existing_secrets_manager_crn` is null."
+}
```

Comment:
This is a DA, should it be hard coded (no option) to `private` or should it at least default to private?

Reply:
addressed